MyCompany.Seeq.Link.Connector.MyConnector/MyConnection.cs

```diff
@@ -154,11 +156,17 @@ public class MyConnection : ISignalPullDatasourceConnection {
             IEnumerator<DatasourceSimulator.Tag> tags = this.datasourceSimulator.Tags;
             while (tags.MoveNext()) {
                 DatasourceSimulator.Tag tag = tags.Current;
+                string tagId = string.Format("{0}", tag.Id);
+
+                this.createChildAsset(rootAssetId, tagId, tag.Name);
```

Comment:
It is a bit strange to me that we obtained a set of tags and we create assets from them. 
I think it would be clearer to separate the two layers.

First I think we should call the simulator and obtain a list of subassets and then for every subasset we get the tags (signals and conditions).